content/cds_hooks.md

```diff
@@ -0,0 +1,117 @@
+---
+title: CDS Hooks
+layout: cds_hooks
+---
+
+# CDS Hooks #
+------------------------------------------------------------------------
+<%= beta_tag(resource: false) %>
+* TOC
+{:toc}
+
+## General ##
+
+### Introduction ###
+The CDS Hooks (Clinical Decision Support Hooks) workflow defines a specification for external CDS Service developers to send notification cards to an electronic health record (EHR). These cards are triggered based on conditions such as the opening of a patient chart of a specified demographic(age, gender, etc.) or when an order is selected. These cards can contain information, a suggestion for the user, or a link to launch a SMART app (See [SMART](./smart) for more information).
```

Comment:
@fpivonka, @VelocityKnight would it be better to reword it as follows?

`The CDS Hooks (Clinical Decision Support Hooks) workflow provides external CDS Service developers the ability to send CDS cards to an electronic health record (EHR). The external CDS service will be triggered based on conditions such as opening a patient chart for the specified demographic(age, gender, etc.) or when an order is selected and the cards returned from the service will be presented to the user in the EHR. The cards can contain textual guidance, actionable suggestion for the user, or a link to launch a SMART app (See [SMART](./smart) for more information).`

Reply:
https://github.com/cerner/fhir.cerner.com/pull/707/commits/49c923c0eb234cfe2f888a5795afc63819dc193e#diff-179aa7a92baef1c89bbd6fabd30f690e85b09c4e22896697d9d13b17dc80d5e7R15